data/korath/korath ships.txt

```diff
@@ -1360,3 +1149,327 @@ ship "Model 512"
 	explode "large explosion" 35
 	"final explode" "final explosion large"
 	description "This is the final state in the evolution of a Kor Mereti war drone: a perfect ring, with incredibly powerful shields and turrets firing in every direction."
+
+
+ship "Korath Ravager"
+	sprite "ship/ravager"
+	thumbnail "thumbnail/ravager"
+	attributes
+		category "Heavy Warship"
+		"cost" 3760000
+		"shields" 27000
+		"hull" 25000
+		"required crew" 319
+		"bunks" 338
+		"mass" 1260
+		"drag" 12.3
+		# Ravager has massive surface area per volume plus heat sinks:
+		"heat dissipation" .8
+		"fuel capacity" 900
+		"cargo space" 110
+		"outfit space" 897
+		"weapon capacity" 415
+		"engine capacity" 232
+		"ramscoop" 2
```

Comment:
Korath have a very good fuel processor outfit. It makes more sense to simply include an extra processor than build it in here. This would give Korath captains a little more flexibility in their outfitting when they are doing shorter range operations. For instance, Korath ships on home-guard duties might value having an extra 14 tons of weaponry at the cost of not having fuel regen and a smaller fuel tanks. If one is sticking with a civilian fleet, after all, let the civilian fleet handle stuff like collecting and storing fuel so the defenders can focus on being more effective defenders.

Reply:
This won't fit with the current outfits unless I make the Dredger's outfit space big enough to eliminate the balance against the Bactrian. I'm adding a bunch more outfits, and I'll resize some. Once I do that, I'll be able to remove built-in ramscoops from most of the ships, including the Dredger.